hoot-core/src/main/cpp/hoot/core/cmd/ChangesetDeriveCmd.cpp

```diff
@@ -88,18 +84,6 @@ class ChangesetDeriveCmd : public BaseCommand
 
     ChangesetCreator(printStats, osmApiDbUrl).create(output, input1, input2);
 
-    if (writeBoundsFile)
-    {
-      ConfigOptions opts;
-      const QString boundsStr = opts.getConvertBoundingBox().trimmed();
-      if (!boundsStr.isEmpty())
-      {
-        OsmMapWriterFactory::write(
-          GeometryUtils::createMapFromBounds(GeometryUtils::envelopeFromConfigString(boundsStr)),
-          opts.getBoundsOutputFile());
-      }
-    }
-
```

Comment:
Looks like you removed this code because it is now in the base class but there is no call to the `_writeBoundsFile` function here.

Reply:
Its in the call to the base class:

https://github.com/ngageoint/hootenanny/blob/568be7baea5fbd91958170f051f0fe7bd7f45ff8/hoot-core/src/main/cpp/hoot/core/cmd/ChangesetDeriveCmd.cpp#L56

Reply:
There it is.  I was expecting it to be called at the end but I guess that isn't necessary.  Looks good.